### ISSUE TITLE
Pretender version release

### ISSUE THREAD

**Issue by zoltanrendes** [samselikoff/ember-cli-mirage#1397]
Hi,

Currently, we have ember-cli-mirage in project's package.json by getting the current master:    
"ember-cli-mirage": "https://github.com/samselikoff/ember-cli-mirage#master" in order to avoid "Cannot read property 'fetch' undefined.
Is that possible to have a release, where pretender has the upgraded version?

**Comment by samselikoff** [comment]
I will cut a release soon!

In the meantime, I would recommend pinning to a specific git sha that contains the code you're after, rather than `master` (since master is a moving target).

For example, the current latest commit is 0213a9573dcf4ef4e7888b1678013d561b6a90fe, so I would put this in your `package.json` for now:

```
"ember-cli-mirage": "samselikoff/ember-cli-mirage#0213a9573dcf4ef4e7888b1678013d561b6a90fe"
```

**Comment by jelhan** [comment]
Should be resolved by `v0.4.10` released on November 30, isn't it?

**Comment by samselikoff** [comment]
Yes! Although I want to come up with a solution to allow folks to change their Pretender versions, as discussed in the Faker issue. I'll consolidate over there.

Thank you 🙏
